charforge-gui/backend/app/api/settings.py

```
@@ -204,8 +217,12 @@ async def test_environment_settings(
         try:
             from google import genai
             client = genai.Client(api_key=google_key)
-            # Simple test - this might need adjustment based on actual API
-            results['GOOGLE_API_KEY'] = {'valid': True, 'message': 'Key format valid'}
+            # Make a minimal API call to verify the key
+            try:
+                models = list(client.list_models())
```

Ruff (0.12.2)

Local variable `models` is assigned to but never used

Remove assignment to unused variable `models`

(F841)